src/box/lua/error.cc

```diff
@@ -192,11 +192,20 @@ luaT_error_create(lua_State *L, int top_base)
 	}
 	struct error *error;
 	error = box_error_new(file, line, code, custom_type, "%s", reason);
-	/*
-	 * Add custom payload fields to the `error' if any.
-	 */
 	if (top_type == LUA_TTABLE) {
+		lua_getfield(L, top_base, "prev");
+		if (!lua_isnil(L, -1)) {
+			struct error *prev = luaL_iserror(L, -1);
+			if (prev == NULL) {
+				luaL_error(L, "Invalid argument 'prev' (error "
+					   "expected, got %s)",
+					   lua_typename(L, lua_type(L, -1)));
+			}
+			/* Cycle is not possible for newly created error. */
+			VERIFY(error_set_prev(error, prev) == 0);
+		}
```

Comment:
I would prefer to group the keyword arguments here: https://github.com/tarantool/tarantool/blob/38fc791cbf50615d6c9a3fb2de23b924449f287c/src/box/lua/error.cc#L171-L177

Reply:
Done.